homework/list.js

```diff
@@ -1,44 +1,37 @@
 'use strict';
 
-(function() {
+(function(exports) {
 
+var ListManager = function() {
   var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+  var _wrapper = null;
+};
 
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
+ListManager.prototype = {
 
-  function onNoteOpen(event) {
+  onNoteOpen(event) {
     if (event.target.classList.contains('note-title')) {
       var id = event.target.dataset.noteId;
-      var content = _listNoteContent[id];
+      var content = this._listNoteContent[id];
       window.dispatchEvent(new CustomEvent('note-open',
         { detail: content }));
     };
-  }
+  },
 
-  function preloadFirstNote() {
-    if (_listNoteContent.length !== 0) {
-      var content = _listNoteContent[0];
+  preloadFirstNote() {
+    if (this._listNoteContent.length !== 0) {
```

Comment:
If a function body is wrapped by a conditional statement only with one branch, it's better to put the `if...else` in the caller, rather than the callee.